rest_framework-stubs/compat.pyi

```diff
@@ -52,3 +52,26 @@ def distinct(queryset: QuerySet, base: Optional[QuerySet]) -> QuerySet: ...
 SHORT_SEPARATORS: Tuple[str, str]
 LONG_SEPARATORS: Tuple[str, str]
 INDENT_SEPARATORS: Tuple[str, str]
+
+__all__ = [
+    "coreapi",
+    "coreschema",
+    "requests",
+    "postgres_fields",
```

Comment:
Since `from django.contrib.postgres import fields as postgres_fields` isn't treated as a re-export, it seems the only way to re-export it is via `__all__`?

Reply:
Looks like it, can we please test it with `tests/typecheck`?